Pipfile

```diff
@@ -59,13 +59,16 @@ zxing-cpp = {version = "*", platform_machine = "== 'x86_64'"}
 # Packages locked due to issues (try to check if these are fixed in a release every so often)
 #
 # Pin this until piwheels is building 1.9 (see https://www.piwheels.org/project/scipy/)
-scipy = "==1.8.1"
+scipy = "~=1.10.0"
```

Comment:
This change breaks armv7

Reply:
Yes, I know, that what I explained here: https://github.com/paperless-ngx/paperless-ngx/pull/3773#issuecomment-1637147614

As you (="paperless people") already discuss about dropping support of armv7, I think it should not be a problem to wait a little bit before merging this PR (I will continue to keep it synced with dev, I use it at home)